### ISSUE TITLE
VerticalStackedBarChart: Accessibility issue in dark mode - axis values are incomprehensible

### ISSUE THREAD

**Issue by tusharjape-msft** [microsoft/fluentui#13894]
### Environment Information

- **Package version(s)**: @uifabric/charting
- **Browser and OS versions**: (fill this out if relevant)

#### Actual behavior:

![dark-fabric-column](https://user-images.githubusercontent.com/66400291/86319066-2bb6e200-bc51-11ea-84af-f6fa3107f4b1.PNG)
In dark mode axis values are incomprehensible

#### Expected behavior:

![dark-highcharts-column](https://user-images.githubusercontent.com/66400291/86319085-3cffee80-bc51-11ea-9169-e6a18bb9249c.PNG)
In dark mode axis values should be comprehensible

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by paulgildea** [comment]
@Raghurk - Mind taking a look at this one? Thanks!

**Comment by Raghurk** [comment]
Looking.

**Comment by Raghurk** [comment]
Please pull the latest charting library and this is fixed
